internal/presenter/packages/model/spdx22/relationship.go

```diff
@@ -2,10 +2,12 @@ package spdx22
 
 type Relationship struct {
 	// SPDX ID for SpdxElement. A related SpdxElement.
-	RelatedSpdxElement string `json:"relatedSpdxElement"`
+	SpdxElementID string `json:"spdxElementId"`
 	// Describes the type of relationship between two SPDX elements.
 	RelationshipType RelationshipType `json:"relationshipType"`
-	Comment          string           `json:"comment,omitempty"`
+	// SPDX ID for SpdxElement.  A related SpdxElement.
```

Comment:
Is this comment a duplicate of https://github.com/anchore/syft/pull/507/files#diff-c37457a5802e2e325dba628b81a3270da8709a215b74f1c421be73c8a60552edR4? Which field should it be used to describe?

Reply:
Yea I'll be more explicit here and use the direct json schema comment for both of these fields

Reply:
updated to most recent specification comments:
https://github.com/spdx/spdx-spec/blob/3d156919a06c07781bd47dc26c75efcfc9df21cd/schemas/spdx-schema.json#L691-L718

Reply:
Better explanation of the transition
https://stackoverflow.com/questions/68360688/whats-the-difference-between-gobuild-and-build-directives

I'm going to remove it for now, but will keep an eye on the transition going forward. I don't know if it's my `LSP` that is doing this or if it's some configuration I updated in my dotfiles when go1.17 dropped which has slipped my mind.

Reply:
Sounds good. And thanks for the links — TIL!

Reply:
Actually, I think we've crossed threads...

Reply:
yea I think I missed the commit when I did a `rebase -i` to fix signature stuff. Addressing now - thanks for following up.

Reply:
I still think I'm not following what this comment is describing, sorry!

Reply:
Here is a link to the spec from their repo:
https://github.com/spdx/spdx-spec/blob/3d156919a06c07781bd47dc26c75efcfc9df21cd/schemas/spdx-schema.json#L697-L711

`spdxElementId`: "Id to which the SPDX element is related"
`relatedSpdxElement`: "SPDX ID for SpdxElement.  A related SpdxElement."

The rest of the model seems to follow these comments or the documentation closely so I was trying to stay in that theme. I do agree it is not the best description.

Reply:
Ah I see. Got it, thanks! 🙏